app/assets/javascripts/discourse/app/mixins/upload-debugging.js

```diff
@@ -0,0 +1,86 @@
+import Mixin from "@ember/object/mixin";
+
+export default Mixin.create({
+  _consoleDebug(msg) {
+    if (this.siteSettings?.enable_upload_debug_mode) {
```

Comment:
Very very minor but in what case would you mix this into something without a `siteSettings`?